orangecontrib/text/widgets/owtweetprofiler.py

```diff
@@ -108,30 +141,45 @@ def set_corpus(self, corpus):
         self.commit()
 
     def apply(self):
-        if self.last_config != self._get_config():
-            self.commit()
+        self.commit()
 
     def _get_config(self):
         return self.tweet_attr, self.model_name, self.output_mode
 
     def commit(self):
         self.Error.clear()
 
+        if self.last_config == self._get_config():
```

Comment:
Those line does not trigger recompute when new data comes. How to reproduce:
Connect data table to corpus and tweet profiler to table in table select different range of data and you will see that recomputation does not happen.

Reply:
Thanks for catching this.

Reply:
Nice solution on this one. :)